backend/Squirrel.ConsoleApp.Models/Models/ErrorCode.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Squirrel.ConsoleApp.Models.Models;
+
+public enum ErrorCode
+{
+    General = 1,
+    FileNotFound,
+    FileDamage
+}
```

Comment:
we already have ErrorType enum in Common project, maybe it is also suitable for this

Reply:
I don't think we need use references from another solution, becouse we need to create self-identifying console app

Reply:
ok